packages/shared/src/lib/utils.shared.ts

```
@@ -16,12 +16,19 @@
   return val.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 };
 
+interface AssertNeverOptions {
```

ESLint

[error] 'AssertNeverOptions' is defined but never used.

(@typescript-eslint/no-unused-vars)